components/CredentialDetailsViews.vue

```diff
@@ -167,10 +170,10 @@ export default {
     const slideNumber = ref(1);
     const tab = ref('highlights');
     const fullscreen = ref(false);
-    const showDetails = ref(false);
     const showDisplays = ref(false);
     const showHighlights = ref(false);
     const credentialImages = reactive([]);
+    const showDetails = ref(config?.vueWallet?.developmentMode);
```

Comment:
Sorry, I've been unclear about this particular feature. What do you think about having this always be enabled (no configuration for it), but clearly marking it as a "Developer feature" or with a "Developer only warning" in the UI / title / display area (above the tree) somewhere?

Reply:
Sounds great - I'll add the warning text and remove the configuration. Thanks for the feedback